src/components/NcDatetimePickerNative/NcDatetimePickerNative.vue

```diff
@@ -249,16 +248,16 @@ export default {
 			}
 			return false
 		},
-		listeners() {
+		attrs() {
 			return {
-				...this.$listeners,
+				...this.$attrs,
 				/**
 				 * Handle the input event
 				 *
 				 * @param {InputEvent} $event input event payload
 				 * @return {Date|string} new chosen Date() or an empty string
 				 */
-				input: ($event) => {
+				onInput: ($event) => {
```

Comment:
If we remove `input` event we can also directly pass `@input="this handler"` and `v-bind="$attrs"` in the template without cloning `$attrs` here.

Reply:
Done.